internal/fields/validate.go

```diff
@@ -448,7 +450,7 @@ func appendECSMappingMultifields(schema []FieldDefinition, prefix string) []Fiel
 				{
 					Name:     "text",
 					Type:     "match_only_text",
-					External: "ecs",
+					External: externalFieldAppendedTag,
```

Comment:
Workaround to be able to filter multifieds generated here during the validation.

Reply:
Would it be a better approach for this ?

Reply:
Do we have many failures if we don't do this? Maybe we can leave this for a separate change.